.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +13,8 @@ Fixes #
 
 **Special notes for your reviewer**:
 
+1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.
```

Comment:
Aren't there cases where a PR could break functionality unless an image is updated? I have seen this happen a couple times already.

Reply:
When we get integration tests, would this complicate things? as in the test will run at HEAD with older images.